xds/src/test/java/io/grpc/xds/internal/certprovider/FileWatcherCertificateProviderTest.java

```diff
@@ -79,6 +80,7 @@
 
   @Rule public TemporaryFolder tempFolder = new TemporaryFolder();
 
+  private long currentTimeMillis = 10_000_000; // arbitrary
```

Comment:
nit: This is not used as a substitute for `System.currentTimeMillis()` but used to set the last-modified timestamp for the files. May be the name (e.g. `lastModifiedTimeToSet`) or the comment can indicate it?

Reply:
Oh, I missed the TimeProvider. The TimeProvider should be swapped to a fake and the files should use the same clock. I'll update things.

Reply:
Fixed. PTAL